modules/apps/portal/portal-monitoring/src/main/java/com/liferay/portal/monitoring/internal/statistics/service/ServerStatisticsHelperUtil.java

```diff
@@ -8,15 +8,12 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
-import org.osgi.service.component.annotations.Component;
-
 /**
  * @author Michael C. Han
  */
-@Component(enabled = false, service = ServerStatisticsHelper.class)
-public class ServerStatisticsHelper {
+public class ServerStatisticsHelperUtil {
```

Comment:
Rename the class to ServerStatisticsUtil and change the commit message so it matches your changes